ci-operator/step-registry/ipi/conf/azure/confidential/confidentialvm/ipi-conf-azure-confidential-confidentialvm-commands.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="/tmp/install-config-security.yaml.patch"
+
+# OCPBUGS-36670: encryptionAtHost and settings.securityType are not related.
```

Comment:
I saw when `ENABLE_CONFIDENTIALVM_COMPUTE` enabled, `encryptionAtHost` will be set to true, which is different with others. Do we do that on purpose ?

Reply:
when enabling `encryptionAtHost` on control plane nodes, the property `encryptionAtHost` on master machine is still null, and I checked that it's set correctly in the spec of capi machine manifest.
```
spec:
  additionalCapabilities:
    ultraSSDEnabled: false
  additionalTags:
    kubernetes.io_cluster.jimatest23a-pzzvm: owned
  allocatePublicIP: true
  diagnostics:
    boot:
      storageAccountType: Managed
  disableExtensionOperations: true
  failureDomain: "2"
  identity: UserAssigned
  image:
    id: /subscriptions/53b8f551-f0fc-4bea-8cba-6d1fefd54c8a/resourceGroups/jimatest23a-pzzvm-rg/providers/Microsoft.Compute/galleries/gallery_jimatest23a_pzzvm/images/jimatest23a-pzzvm-gen2
  networkInterfaces:
  - privateIPConfigs: 1
    subnetName: jimatest23a-pzzvm-master-subnet
  osDisk:
    cachingType: ReadWrite
    diskSizeGB: 1024
    managedDisk:
      securityProfile:
        securityEncryptionType: VMGuestStateOnly
      storageAccountType: Premium_LRS
    osType: Linux
  providerID: azure:///subscriptions/53b8f551-f0fc-4bea-8cba-6d1fefd54c8a/resourceGroups/jimatest23a-pzzvm-rg/providers/Microsoft.Compute/virtualMachines/jimatest23a-pzzvm-bootstrap
  securityProfile:
    encryptionAtHost: true
    securityType: ConfidentialVM
    uefiSettings:
      secureBootEnabled: true
      vTpmEnabled: true
```
looks like some issues in upstream, because same setting works well on compute nodes (BTW, encryptionAtHost is not allowed to be enabled when osdisk security encryption type is DiskWithVMGuestState, installer has pre-check for it), will try terraform way and check with dev later.

So only enable `ENABLE_CONFIDENTIALVM_COMPUTE` here, this setting can also cover checkpoint for OCPBUGS-36670.